src/lib/tools.ts

```diff
@@ -1,39 +1,65 @@
 'use strict';
```

Comment:
No need for this in a TS file.

Reply:
Thanks for spotting! (missed when copying some js file imports...)